docs/api/covidcast-signals/safegraph.md

```diff
@@ -131,7 +131,7 @@ COVIDcast API.
 
 ## Limitations
 
-SafeGraph's Social Distancing Metrics and Weekly Patterns are based on mobile devices that are members of SafeGraph panels, which is not necessarily the same thing as measuring the general public. These counts do not represent absolute counts, and only count visits by members of the panel in that region. This can result in several biases:
+SafeGraph's [Social Distancing Metrics](https://docs.safegraph.com/docs/social-distancing-metrics) and [Weekly Patterns](https://docs.safegraph.com/docs/weekly-patterns) are based on mobile devices that are members of SafeGraph panels, which is not necessarily the same thing as measuring the general public. These counts do not represent absolute counts, and only count visits by members of the panel in that region. This can result in several biases:
```

Comment:
```suggestion
SafeGraph's [Social Distancing Metrics](https://docs.safegraph.com/docs/social-distancing-metrics) 
and [Weekly Patterns](https://docs.safegraph.com/docs/weekly-patterns) data products
are based on mobile devices that are members of SafeGraph panels, which is not necessarily the same thing as measuring the general public. 
These counts do not represent absolute counts, and only count visits by members of the panel in that region. This can result in several biases:
```